helm_deploy/values-prod.yaml

```diff
@@ -69,7 +69,7 @@ generic-service:
     FEATURE_EVENT_PERSONALRELATIONSHIPS_PERSON_IDENTIFIERS-UPDATED: false
     FEATURE_EVENT_PERSONALRELATIONSHIPS_PERSON_IDENTIFIERS-DELETED: false
     FEATURE_EVENT_PERSONALRELATIONSHIPS_OFFENDER_PHYSICAL_DETAILS-CHANGED: false
-    FEATURE_EVENT_PERSONALRELATIONSHIPS_PRISON-OFFENDER-EVENTS_PRISONER_BOOKING_MOVED: false
+    FEATURE_EVENT_PERSONALRELATIONSHIPS_PROFILEDETAILS_PRISON-OFFENDER-EVENTS_PRISONER_BOOKING_MOVED: false
```

Comment:
Renamed the feature switch type to indicate this is profile details

Reply:
Looking at `isEnabled(eventType, "personalrelationshipsprofiledetails") `, should it be
`FEATURE_EVENT_PERSONALRELATIONSHIPSPROFILEDETAILS_PRISON-OFFENDER-EVENTS_PRISONER_BOOKING_MOVED`